telemetry/otel-collector-config.yaml

```diff
@@ -1,9 +1,29 @@
+# https://opentelemetry.io/docs/collector/configuration/
+
+# How data gets into the Collector
 receivers:
+  # Data sources: traces, metrics, logs
   otlp:
     protocols:
+      http:
```

Comment:
Please omit this as it is unused. We export traces in `otlp` format over the `grpc` protocol. We do not currently have any additional configuration that we can provide to switch the protocol from `grpc` to `http`.

Reply:
Ah! You are right..